Gruntfile.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function (grunt) {
+
+    "use strict";
+
+    grunt.initConfig({
+        buster: {
+            all : {
+                test: {
+                    config: "test/buster.js"
+                },
+                server: {
+                    port: 1112
+                }
+            }
+        }
+    });
+
+    grunt.loadNpmTasks("grunt-buster");
+
+    grunt.registerTask("test", ["buster"]);
+    grunt.registerTask("default", "test");
+};
```

Comment:
I don't get this whole grunt thing. I'd personally prefer [using plain npm scripts as a build tool](http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/), but maybe you can convince me?